README.md

```diff
@@ -1,2 +1,33 @@
-# loopback-sdk-ios-codegen
-Code generation tool for LoopBack iOS SDK
+# LoopBack iOS SDK CodeGen
+
+The iOS CodeGen command line tool generates iOS client side LoopBack Model representations
+in Objective-C by looking into the specified server application.
+
+## How to Use
+
+The following is an example of the usage applied to the test server provided under this directory:
+
+ 1. Run `npm install` in this directory to initialize the tool `bin/lb-ios` and the test server under `test-env/server`.
+ *  Run `bin/lb-ios -p XX test-env/server/server test-env/client/ios/gen-src` 
+ 	to generate the models in Objective-C representation, 
+ 	where `XX` is the prefix attached to all the generated class names.
+
+## How to Run the Unit Test
+
+After performing the above steps, run the followings:
+
+ 1. Run the test server by `node test-env/server/server.js &`.
+ *  Copy `LoopBack.framework` into `test-env/CodeGenTest/`.
```

Comment:
Sure, I'll update the usage guide to include where to get the framework.
Since I needed some updates in the SDK to make the codegen work fully, the framework needs to come from  https://github.com/strongloop/loopback-sdk-ios.  I'll add this to the README.